src/Stryker.Core/Stryker.Core/Initialisation/InitialisationProcess.cs

```diff
@@ -33,12 +35,18 @@ public class InitialisationProcess : IInitialisationProcess
             _reporter = reporter;
             _inputFileResolver = inputFileResolver ?? new InputFileResolver();
             _initialBuildProcess = initialBuildProcess ?? new InitialBuildProcess();
-            _initialTestProcess = initialTestProcess ?? new InitialTestProcess();
+            _initialTestProcess = initialTestProcess ?? CreateInitionTestProcess();
             _testRunner = testRunner;
             _logger = ApplicationLogging.LoggerFactory.CreateLogger<InitialisationProcess>();
             _assemblyReferenceResolver = assemblyReferenceResolver ?? new AssemblyReferenceResolver();
         }
 
+        private static InitialTestProcess CreateInitionTestProcess()
```

Comment:
Should this not be CreateInitialTestProcess?

Reply:
Renamed